fluentd/configs.d/openshift/filter-pre-k8s-audit.conf

```diff
@@ -0,0 +1,6 @@
+<filter audit.log>
+  @type record_transformer
+  <record>
+    time ${time}
```

Comment:
I think you want the audit log parser to add either the `time` or the `@timestamp` field

Reply:
isn't that what this configuration does?

the audit log parser is meant for auditd logs, not kubernetes ones :/

Reply:
I'm confused - are you using the tag `audit.log` for both auditd logs and kubernetes audit logs?

Reply:
ok - try this
```
   @type record_transformer
   enable_ruby true
   <record>
     time "${record['requestReceivedTimestamp'] || record['time']}"
   </record>
```
that way it should work with either kind of record tagged with `audit.log`

Reply:
Thanks! I took the feedback and added to the CLO PR https://github.com/jcantrill/cluster-logging-operator/pull/1